src/state/actions/dispatchArIOInteraction.ts

```diff
@@ -58,8 +58,16 @@ export default async function dispatchArIOInteraction({
         let antProcessId: string = payload.processId;
 
         if (antProcessId === 'atomic') {
+          const state =
+            payload.state ||
+            (payload.targetId
+              ? createAntStateForOwner(
+                  owner.toString(),
+                  payload?.targetId?.toString(),
+                )
+              : undefined);
           antProcessId = await spawnANT({
-            state: payload.state,
+            state,
```

Comment:
Do we ever want to spawn an ant where state may be undefined?

Reply:
Yup